src/components/Posts/Posts.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+import Post from "./Post";
+import { PostContainer } from "./Post.style";
+
+const Posts = (props) => {
+  const [Posts, usePosts] = useState([]);
+  const userId = props.user;
+  console.log(userId);
+
+  useEffect(() => {
+    (async () => {
+      const userPosts = await fetchPosts();
+      // eslint-disable-next-line react-hooks/rules-of-hooks
+      usePosts(userPosts);
+    })();
+  }, []);
+
+  const fetchPosts = async () => {
+    const result = await fetch(
+      `https://jsonplaceholder.typicode.com/posts?userId=${userId}`
+    );
+    const data = await result.json();
+    return data;
+  };
+  return (
+    <PostContainer>
+      {Posts.length > 0
+        ? Posts.map((post) => <Post post={post} />)
+        : "no posts to show"}
+    </PostContainer>
```

Comment:
this will cause the "no posts to show" while the data is loading, handle the loading state
the component will have 3 different states, no data, have data and loading data; which means we still don't know if we have data to show or not